automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -636,10 +673,10 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     val transformed = for {
       stagingBucketName <- OptionT.fromOption[Future](cluster.stagingBucket)
       stagingBucketObjects <- OptionT.liftF[Future, List[GcsObjectName]](googleStorageDAO.listObjectsWithPrefix(stagingBucketName, "google-cloud-dataproc-metainfo"))
-      initLogFile <- OptionT.fromOption[Future](stagingBucketObjects.filter(_.value.endsWith("dataproc-initialization-script-0_output")).headOption)
+      initLogFile <- OptionT.fromOption[Future](stagingBucketObjects.find(_.value.endsWith("dataproc-initialization-script-0_output")))
```

Comment:
No logic change above and below...just combined `filter` + `headOption` into `find` for conciseness.